guides/common/modules/proc_configuring-dynamic-dns-update-with-gss-tsig-authentication.adoc

```diff
@@ -113,7 +113,7 @@ For example, `example.com`.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-grant {smart-proxy-context}/047__{foreman-example-com}@EXAMPLE.COM__ wildcard * ANY;
+grant foremanproxy\047__{foreman-example-com}@EXAMPLE.COM__ wildcard * ANY;
```

Comment:
I don't think this is correct in itself. On line 101/102 the same principal is used to initialize it and it's used elsewhere too. I do wonder why there's 047 in here but not in the others.